### ISSUE TITLE
Remove mouse/pad inputs from counters

### ISSUE THREAD

**Issue by henricazottes** [RoanH/KeysPerSecond#17]
Hi, awesome work!

I wanted a soft to check how many keys I press during a day as a developer. KPS is exactly what I needed ;)

When tracking all keys, it seems to also track clicks, would it be possible to exclude mouse events from counters? or maybe add a specific counter for that with number of click, distance scrolled, distance moved, etc...

**Comment by RoanH** [comment]
Hey,

Glad to hear that the program is useful to you.
You're right that tracking all mouse clicks and key presses while the setting is called "Track all keys" is a bit counter intuitive. That is definitely something I overlooked when adding support for mouse buttons.

For version 8.2 I am going to split the option into two separate options, one to track the mouse button presses and one to track the key presses.

More specific and customizable counters are also items on my long term todo list. 
I did also start working on other mouse statistics, though I intended to release that as a separate program sometime in the future. Since it feels a bit out of scope for this program.

I'm currently a bit busy, but I should be able to complete this release within 1-3 weeks.
